<launch>

  <rosparam command="load" file="$(env HOME)/workspace/data/cams/v1/cam0/sensor.yaml" ns="/cam0"/>
  <rosparam command="load" file="$(env HOME)/workspace/data/cams/v1/cam1/sensor.yaml" ns="/cam1"/>

  <node name="stereo_rectifier_node" pkg="dsvo" type="stereo_rectifier_node">
    <param name="cam0_topic" value="/cam0/image_raw" />
    <param name="cam1_topic" value="/cam1/image_raw" />
  </node>

  <node name="stereo_processor_node" pkg="dsvo" type="stereo_processor_node" output="screen" />

  <node name="comparer_node" pkg="dsvo" type="comparer_node" args="trans" />

  <node name="dynamic_reconfigure_load" pkg="dynamic_reconfigure" type="dynparam" args="load /stereo_processor_node $(find dsvo)/cfg/v.yaml" />

</launch>
